scripts/fetch/addToTable.R

```diff
@@ -0,0 +1,7 @@
+library(yaml)
```

Comment:
We'll want to be careful that this doesn't get sourced, as it could overwrite edits that have been made.